### PR TITLE
servlets/task1 - clicked link counter

### PR DESCRIPTION
**Pull request by qnislavnachev**
there is a link click counter implemented by servlet with get method

### CODE REVIEW

**Comment by qnislavnachev** [comment]
link counter needs a review

**Comment by qnislavnachev** [comment]
updated counter servlet now we can add additional links and render method get resource file as stream

File: src/test/java/counterservlet/CountingServletTest.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            oneOf(request).getParameter("link");
+            will(returnValue(null));
+
+            oneOf(response).getWriter();
+            will(returnValue(writer));
+        }});
+
+        countingServlet.doGet(request, response);
+
+        assertThat(out.toString().contains("I am link 1 - Clicked: 5"), is(equalTo(true)));
```
Comment (mgenov) [inline src/test/java/counterservlet/CountingServletTest.java]: Doesn't look correct as init parameters of the sessions are with the same numbers: 5, 1, 2. So after click counters are not updated.

File: src/main/java/counterservlet/Template.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package counterservlet;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Template {
```
Comment (mgenov) [inline src/main/java/counterservlet/Template.java]: public final class Template

**Comment by qnislavnachev** [comment]
fixed test renderPage after click attributes get updated

**Pull request merged**